Assets/Scripts/Player/WeaponInventory.cs

```diff
@@ -190,8 +219,21 @@ private void RemoveWeapon(int weaponIndex)
             state = State.NO_WEAPONS;
         }
 
+        PickupableItem dropData = Instantiate(Pickupable, gameObject.transform.position, new Quaternion(0, 0, 0, 0))
+                                      .GetComponent<PickupableItem>();
+        dropData.Weapon = Weapons[weaponIndex];
+        dropData.WeaponEffect = weaponItems[weaponIndex];
+
+        weaponItems.RemoveAt(weaponIndex);
         Weapons.RemoveAt(weaponIndex);
         weaponAmmoAmounts.RemoveAt(weaponIndex);
-        ChangeWeapon(0, true);
+        ChangeWeapon(0, true, true);
+    }
+
+    private void SelectItem()
+    {
+        GameObject[] choices = Resources.LoadAll<GameObject>("Item/Weapon");

```

Comment:
Are we getting item options here or weapons?

Reply:
The other types are pickupable and fragmentation